src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/ManagedDebuggerImageTypeService.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug
+{
+    /// <summary>
+    ///     Provides debuggers with information agbout the target output that the project generates.
+    /// </summary>
+    [Export(typeof(IDebuggerImageTypeService))]
+    [AppliesTo(ProjectCapability.CSharpOrVisualBasic)]
+    internal class ManagedDebuggerImageTypeService : IDebuggerImageTypeService
+    {
+        private readonly ProjectProperties _properties;
+
+        [ImportingConstructor]
+        public ManagedDebuggerImageTypeService(ProjectProperties properties)
+        {
+            Requires.NotNull(properties, nameof(properties));
+
+            _properties = properties;
+        }
+
+        public ImageClrType TargetImageClrType
+        {
```

Comment:
Although Expression bodies are not part of coding guidelines, should we include them?

Sorry, I am a little late to party :(